README.md

```diff
@@ -1317,6 +1317,9 @@ Other Style Guides
     numbers.forEach(num => sum += num);
     sum === 15;
 
+    const sum = numbers.map(num => num + 1);
```

Comment:
I'm not sure what the benefit of this example is - are you implying someone might try to use `map` to add up the numbers in an array? This will produce an array of `[2, 3, 4, 5, 6]`, which is not `=== 15`.